Client/Clique/Modules/CreatedEvents/CreatedEventsViewController.swift

```diff
@@ -8,7 +8,72 @@
 import UIKit
 
 class CreatedEventsViewController: UIViewController {
+    
+    @IBOutlet private var _tableView : UITableView?
+    
+    private var _events : [Event] = [] {
```

Comment:
```suggestion
    private var _events: [Event] = [] {
```

formatting, apply this everywhere